plugins/modules/aci_epg.py

```diff
@@ -163,7 +163,7 @@
     host: apic
     username: admin
     password: SomeSecretPassword
-    validate_certs: no
+    validate_certs: false
```

Comment:
in line 122 there is also a mention of preferred_group: yes (mentioning like this because I cant comment on that line)